libs/client/src/lib.rs

```diff
@@ -24,6 +24,7 @@ use std::os::unix::fs::MetadataExt;
 use std::path::Path;
 use std::time::Duration;
 
+use kanidm_proto::constants::APPLICATION_JSON;
```

Comment:
Is there not a version of this header in reqwest?

Reply:
Nope, I looked around trying to find it 

* there's one in `mime` since it's a header value rather than a header key, but we don't currently use this lib, and it's a bit weird from memory
* I checked we were already building `kanidm_proto` in the libs where I added this, so  I'm pretty sure it's not adding to the mess much